### DISCUSSION TITLE
No more information available ?

### DISCUSSION THREAD

**Discussion by RuudSchmeitz** [pawelsalawa/sqlitestudio#4107] [Q&A]
In the process of **modifying the constraints of a column** , I clicked on the question mark in the title bar of any of these windows for more information :

![ModColumn06](https://user-images.githubusercontent.com/69636360/117914011-bf7e2380-b2e2-11eb-829b-c28d25e1b019.png)

but nothing happened .

Where can I find this information ?

**Comment by pawelsalawa** [comment]
I think they are there by default, but I didn't populate them with any information. I guess I need to find a way to remove them from title bar.

If you need help with usage, you can go to the https://github.com/pawelsalawa/sqlitestudio/wiki - there's the user manual. It's not 100% complete, but maybe it will be useful for you.

**Comment by RuudSchmeitz** [comment]
Just some minor points :

In the **User_Manual** :
https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual
there are 4 links for `Configuration Dialog` .
The first 2 work just fine :
https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#configuration-dialog
but the last 2 don't :
https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_dialog

In the **Tips_&_Tricks** :
https://github.com/pawelsalawa/sqlitestudio/wiki/Tips_&_Tricks
there are these 3 links for `SQL Editor` :
https://github.com/pawelsalawa/sqlitestudio/wiki/SQL_Editor
Is that the correct link ?
Or should it be :
https://github.com/pawelsalawa/sqlitestudio/wiki/User_Manual#sql-editor-window

**Reply by pawelsalawa** [reply]
Fixed. Thank you for spotting these!

I have migrated the wiki some time ago from hosted MediaWiki into GitHub wiki and I had to adjust links. I did it with automated script, but apparently it did not catch all the links correctly.
